File: code-coaches/medium/deja-vu/DejaVu.cs
using System;

public class DejaVu
{
    static void CheckDejaVu(string chars)
    {
        char previousChar = ' ';
        bool isRepeated = false;

        for (int i = 0; i < chars.Length; i++)
        {
            char c = chars[i];

            if (c == previousChar)
            {
                Console.WriteLine("Deja Vu");
                isRepeated = true;
                break;
            }
            else
            {
                previousChar = c;
            }
        }

        if (!isRepeated)
        {
            Console.WriteLine("Unique");
        }
    }

    static void Main(string[] args)
    {
        string chars = Console.ReadLine();
        CheckDejaVu(chars);
    }
}

File: code-coaches/medium/deja-vu/DejaVu.swift
func checkDejaVu(chars: String) {
    var previousChar: Character?
	var isRepeated = false

	for char in chars {
		if char == previousChar {
			print("Deja Vu")
			isRepeated = true
			break
		} else {
			previousChar = char
		}
	}

	if !isRepeated {
			print("Unique")
	}
}

func main() {
    let chars = readLine()!
	checkDejaVu(chars: chars)
}

main()